models/cow_protocol/ethereum/cow_protocol_ethereum_trades.sql

```diff
@@ -226,4 +226,6 @@ valued_trades as (
         ON trades.order_uid = efs.order_uid
 )
 
-select * from valued_trades
+select *,
+  ((limit_sell_amount - atoms_sold) / atoms_sold  * usd_value) + ((atoms_bought - limit_buy_amount) / atoms_bought * usd_value) as surplus_usd
```

Comment:
Do you have a link to a query showing that this works? I thought we would have to perform the calculation differently depending on whether it is a buy or sell order. I assume that one of these two terms is zero and that is why there is no need for a case statement (could we put a comment here for that).